docker-compose/node-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# Ensure that expected dirs exist.
+mkdir -p "${CONCORDIUM_NODE_CONFIG_DIR}" "${CONCORDIUM_NODE_DATA_DIR}"
+
+if [ -n "${GENESIS_DATA_PATH}" ]; then
+	# Copy 'genesis.dat' - better solution: support flag in concordium-node to set location.
+	cp "${GENESIS_DATA_PATH}/genesis.dat" "${CONCORDIUM_NODE_DATA_DIR}"
+	# Select unique baker credentials file.
+	id="$(curl "http://${BAKER_ID_GEN_DNS}:8000/next_id")"
+	export CONCORDIUM_NODE_BAKER_CREDENTIALS_FILE="${GENESIS_DATA_PATH}/bakers/baker-${id}-credentials.json"
+fi
```

Comment:
What is the intend of the `GENESIS_DATA_PATH` variable? What if it is not set, is the assumption that the data dir already contains genesis.dat? If not, I think the script should fail directly here.

Could you document the intended behaviour please.

Reply:
Added some docs to explain the setup. I think that since the node will fail by itself if `genesis.dat` is shouldn't be necessary to do that same check here.

Note that I don't expect users to modify the Compose file. But if they decide to remove `GENESIS_DATA_PATH` it must be because they want to take control of the setup just like if they change some other setting that we also don't check. In that case I think it's best for the scripts to get out of the way and interfere as little as possible.